src/test/kotlin/calculator/CalculateTypeTest.kt

```diff
@@ -0,0 +1,54 @@
+package calculator
+
+import org.assertj.core.api.Assertions.assertThat
+import org.assertj.core.api.Assertions.assertThatExceptionOfType
+import org.junit.jupiter.api.Assertions.assertTrue
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.params.ParameterizedTest
+import org.junit.jupiter.params.provider.ValueSource
+import java.lang.IllegalArgumentException
+
+class CalculateTypeTest {
+
+    @ParameterizedTest
+    @ValueSource(strings = ["+", "-", "*", "/"])
+    fun is_valid_symbol(input: String) {
```

Comment:
코틀린에서는 아래와 같이 공백을 함수명에 넣을 수도 있습니다.
```suggestion
    fun `is valid symbol`(input: String) {
```